sdk/core/core-rest-pipeline/src/policies/setClientRequestIdPolicy.ts

```diff
@@ -9,6 +9,29 @@ import { PipelinePolicy } from "../pipeline";
  */
 export const setClientRequestIdPolicyName = "setClientRequestIdPolicy";
 
+/**
+ * Options for how HTTP connections generate the client request id header
+ * requests.
+ */
+export interface SetClientRequestIdPolicyOptions {
+  /**
+   * The header name to use for the client request id
+   */
+  requestIdHeaderName?: string;
+  /**
+   * Whether or not to include a the generateClientRequestPolicyId pipeline
+   */
+  disable?: boolean;
+}
+
+/**
+ * Default options for ClientRequestId policy
+ */
+export const DefaultSetClientRequestIdPolicyOptions: SetClientRequestIdPolicyOptions = {
```

Comment:
feels bad that this options type *isn't* what `setClientRequestIdPolicy` takes as options. This is most of why I think `disable` booleans are bad.